cypress/fixtures/blogArticles.js

```diff
@@ -0,0 +1,220 @@
+[{
```

Comment:
`.js` but is a JSON file. Change extension or export the array.

Reply:
You're right. It happened from c/p from json-server, used node cli to convert it and didn't change the right format afterwards :x